src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -68,6 +70,19 @@ internal class ManifestDocument
 				doc.Root.SetAttributeValue (androidNs + "versionCode", value);
 			}
 		}
+		public string MinimumSdk {
```

Comment:
Perhaps this should be a `GetMinimumSdk()` method, not a property, given that this property isn't just a field access? It would inform callers that they may want to cache the value instead of calling the property repeatedly.